analytics/management/commands/update_page_analytics.py

```diff
@@ -75,13 +75,13 @@ def handle(self, *args, **options):
                 '{} must be a file which contains your api key.'.format(key_file_location)
             )
 
-        scope = ['https://www.googleapis.com/auth/analytics.readonly']
+        scopes = ['https://www.googleapis.com/auth/analytics.readonly',]
```

Comment:
The trailing comma is important with tuples so Python know the difference between a tuple of 1 ('foo',) and a value in parentheses ('foo') * 6, but it isn't required with lists. This common also failed flake8 validation so the integration tests couldn't be completed. Remove the comma a give it another go.